tests/test_kernels.py

```diff
@@ -77,6 +77,28 @@ def _ref_periodic(X, lengthScale, signal_variance, period):
     return signal_variance * exp_dist
 
 
+def _ref_changepoints(X, kernels, locations, steepness):
+    # Calculates K(X) for each kernel in `kernels`, then multiply by sigmoid functions
```

Comment:
Can you make it as a documentation with """...""". Thanks!

Reply:
Done